docs/detections/prebuilt-rules/prebuilt-rules-downloadable-updates.asciidoc

```diff
@@ -0,0 +1,19 @@
+[[prebuilt-rules-downloadable-updates]]
+[role="xpack"]
+== Downloadable rule updates
+
+This section lists all updates to prebuilt detection rules, made available with the *Prebuilt Security Detection Rules* integration in Fleet.
+
+To download the latest updates, follow the instructions in <<download-prebuilt-rules>>
+
+
+[width="100%",options="header"]
+|==============================================
+|Update version |Date | New rules | Updated rules
+
+|<<prebuilt-rule-0-13-1-prebuilt-rules-0-13-1-summary, 0.13.1>> | 21 Jun 2021 | 4 | 41
+
+|==============================================
+
+
+include::downloadable-packages/0-13-1/prebuilt-rules-0-13-1-summary.asciidoc[]
```

Comment:
If you want to bump pages down a level in the navigation tree (without having to edit the === heading level in each one), you can use the [leveloffset attribute](https://docs.asciidoctor.org/asciidoc/latest/directives/include-with-leveloffset/#manipulate-heading-levels-with-leveloffset) like this:
```suggestion
include::downloadable-packages/0-13-1/prebuilt-rules-0-13-1-summary.asciidoc[leveloffset=+1]
```

Reply:
ok perfect - this may be exactly what I need, thanks!